.chronus/changes/azhang_RemoveClientFlatten-2024-6-1-16-58-12.md

```diff
@@ -0,0 +1,8 @@
+---
+changeKind: breaking
+packages:
+  - "@azure-tools/typespec-autorest"
+  - "@azure-tools/typespec-azure-resource-manager"
+---
+
+x-ms-client-flatten extension on some of resource properties property is now configurable to be emitted by autorest emitter. Default is false which will skip emission of that extension.
```

Comment:
```suggestion
`x-ms-client-flatten `extension on some of resource properties property is now configurable to be emitted by autorest emitter. Default is false which will skip emission of that extension.
```

can you also describe what is breaking and how one would fix it as its not clear to me from this text